point_viewer_proto_rust/src/proto.proto

```diff
@@ -75,23 +75,47 @@ message NodeId {
   int64 deprecated_index = 2;
 }
 
-message Node {
-  enum PositionEncoding {
+enum PositionEncoding {
     INVALID = 0;
     Uint8 = 1;
     Uint16 = 2;
     Float32 = 3;
     Float64 = 4;
-  }
+}
 
+message Node {
   PositionEncoding position_encoding = 2;
   int64 num_points = 3;
   NodeId id = 4;
 }
 
+enum AttributeDataType {
+    INVALID_DATA_TYPE = 0;
```

Comment:
Nit: `INVALID` would be enough...

Reply:
Can you explain why this is needed?

Reply:
See https://github.com/googlecartographer/point_cloud_viewer/pull/309#discussion_r308626031

Reply:
it is needed by proto for value 0, and the by compiling it complained having 2 INVALID in the same file (which i also found curious though